common/app/templates/inlineJS/blocking/applyRenderConditions.scala.js

```diff
@@ -70,6 +71,12 @@
         docClass += ' has-fixed';
     }
 
+    if (supportsSupports && testCssSupportForPropertyAndValue('position', 'sticky')) {
```

Comment:
> is the idea that you won't have sticky if you don't have supports?

Exactly. I think you're right that it's confusing, especially compared to the `fixed` test just above. Plus if for some reason `window.CSS` is not available in a supporting browser, the test will fail.